### ISSUE TITLE
How TRUST4 defines clones

### ISSUE THREAD

**Issue by resendejss** [liulab-dfci/TRUST4#238]
Congratulations on the tool.

I'm new to the area, excuse my ignorance.

I ran the TRUST4 tool on tumor FASTQ data and obtained the CDR3 sequences. I am using these results in Immunarch and in the process, clones are taken into account (for BCR same gene V and J same length of CDR3 and 90% identity; for TCR same gene V and J, same length of CDR3 and 95% identity ). In this case, can I pass the TRUST4 results into this clone process?

Or is TRUST4 already considering clones with this metric?

**Comment by mourisl** [comment]
Yes, I believe you can directly pass it to Immunarch. The raw output from TRUST4 does not group CDR3s.

**Comment by resendejss** [comment]
Thanks
